ui/src/components/integrationFormFields/integrationFormFields.jsx

```diff
@@ -1,11 +1,11 @@
 import React, { useEffect } from 'react';
-import { LABELS } from '../constans';
+import { LABELS } from '../constants';
 
 export const IntegrationFormFields = (props) => {
   const { initialize, disabled, initialData, updateMetaData, ...extensionProps } = props;
   const {
     components: { FieldErrorHint, FieldElement, FieldText, FieldTextFlex },
-    validators: { requiredField, btsUrl, btsProjectKey, btsIntegrationName, email },
+    validators: { requiredField, btsJiraCloudUrl, btsProjectKey, btsIntegrationName, email },
```

Comment:
This will break the backward compatibility.
The changes done in develop will be released in the next year, but the plugin should work with RP-24.2.2 and older versions.
So, please create the necessary validator right here in plugin.
You can check custom validations in QG plugin as a reference.

Reply:
Done.